### PR TITLE
[QE] cpu version with gcc

### PR DESCRIPTION
**Pull request by toxa81**
ReFrame tested UENV stack for CPU and GPU versions of QE.

### CODE REVIEW

**Comment by toxa81** [comment]
update: with this current setup of the test ReFrame can't capture the end of job. Pending forever. 
```
[2023-08-30T22:33:36] debug2: reframe: Current tests: 1
[2023-08-30T22:33:36] debug: QuantumESPRESSOCpuCheck %scale=small /ab356e35 @hohgant:cpu+quantumespresso_default: Entering stage: run
[2023-08-30T22:33:36] debug2: QuantumESPRESSOCpuCheck %scale=small /ab356e35 @hohgant:cpu+quantumespresso_default: Selecting subconfig for 'hohgant:cpu'
[2023-08-30T22:33:36] debug2: QuantumESPRESSOCpuCheck %scale=small /ab356e35 @hohgant:cpu+quantumespresso_default: Selecting subconfig for 'hohgant'
[2023-08-30T22:33:36] debug2: reframe: Bumped 0 test(s)
[2023-08-30T22:33:36] debug2: reframe: Poll rate control: sleeping for 10s (current poll rate: 0.19809336138296135 polls/s)
```

The job has finished successfully, I checked that.

File: recipes/quantumespresso/cpu/config.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: quantumespresso
+store: /user-environment
+description: QuantumESPRESSO application and dependencies.
```
Comment (bcumming) [inline recipes/quantumespresso/cpu/config.yaml:3]: `QuantumESPRESSSO application and dependencies for multicore nodes.`

File: recipes/quantumespresso/ci.yml [hunk 2]
```diff
@@ -1,7 +1,18 @@
 include: '/ci/common.yml'
 
-build-a100:
-  extends: .stack-build-hohgant-a100
+#build-a100:
```
Comment (bcumming) [inline recipes/quantumespresso/ci.yml:3]: Don't forget to uncomment this.

Will the GPU ReFrame tests be added in a separate PR?

**Review by bcumming — CHANGES_REQUESTED** [review]

File: recipes/quantumespresso/ci.yml (hunk 2 above)
Comment (toxa81) [reply recipes/quantumespresso/ci.yml:3]: Yes, I want to test CPUs first and be ready for Eiger.

**Comment by toxa81** [comment]
@bcumming this is ready

**Review by bcumming — APPROVED** [review]
